lib/Babel.js

```diff
@@ -83,6 +84,12 @@ Babel.prototype.getAsync = function getAsync(opts) {
     switch (opts.type) {
       case undefined:
       case 'tile': {
+        // return tile as-is
+        if (self.pickLanguage &&
+          (opts.lang || self.defaultLanguage) === 'local') {
```

Comment:
Is defaultLanguage overriding lang, if both are given?

If so, please reflect that in the test (instead of writing some random nonexisting language, add in an existing language and make sure defaultLanguage overrides) -- if not, then the condition should be something like if we don't have opts.lang, check defaultLanguage, and then if either is local, continue the condition.

Reply:
defaultLanguage is only considered is lang is not provided.

The tests have been updated to illustrate both cases. Also, I've taken the liberty to rename lng and rlang to what they actually are, I hope you won't mind.